docs/src/content/docs/guides/i18n.mdx

```diff
@@ -56,6 +61,7 @@ Starlight provides built-in support for multilingual sites, including routing, f
          - ar/
          - en/
          - zh/
+         - ko/
```

Comment:
Similarly here, I think we can leave this example unchanged and people will still understand that they could use a `ko/` directory for Korean.
```suggestion
```